test/unit_tests/aievec_tests/i16xi16_static_sized_memref_aie-ml/conv2d_uij_i16_noinit.mlir

```diff
@@ -0,0 +1,70 @@
+// REQUIRES: valid_xchess_license
+// RUN: aie-opt %s -affine-super-vectorize="virtual-vector-size=16" -aieml=true --aie-vectorize="shift=10 zero-offset=4" | aie-translate -aieml=true --aievec-to-cpp -o gen_aie-ml.cc
+// RUN: xchesscc -f -g +s -p me -P ${XILINX_VITIS}/aietools/data/aie_ml/lib/ +w work +o work -I%S -I. %S/testbench.cc %S/kernel.cc
```

Comment:
Notice that `XILINX_VITIS` is not defined in our "lit" config files. If you're running this test on its own on your computer, it will pick it up from your environment, but `lit` will wipe out before running. I'm already tripping over another test that does this, if I run `ninja check-aie` it warns me that `XILINX_VITIS` is being unset and proceeds to fail.

There is a substitution that replaces "aietools" (defined as `-DVITIS_AIETOOLS_DIR` on cmake configuration) with what I'd expect "${XILINX_VITIS}/aietools" to point at, so you should probably replace what you have there with:

```bash
xchesscc -f -g +s -p me -P %aietools/data/aie_ml/lib/ +w work +o work -I%S -I. %S/testbench.cc %S/kernel.cc
```